docs/initial-milestones.md

```diff
@@ -0,0 +1,65 @@
+# OpenTelemetry: A Roadmap to Convergence
+
+This document covers the intial milestones for each SIG developing an implementation of the OpenTelemetry Specification. For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
+
+[Merging OpenTracing and OpenCensus: A Roadmap to Convergence](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289) defines the goals and timelines for OpenTelemetry.
+
+If you are interested in starting a new SIG, please do the following:
+* [create an issue](github.com/open-telemetry/community/issues) proposing the new SIG.
+* [Configure the new repository](docs/how-to-configure-new-repository.md).
+* Add the milestones listed below to the backlog.
+
+## Initial Milestones
+For languages which have both an OpenTracing and OpenCensus implementation, we would like to achieve parity in OpenTelemetry by **September, 2019**, and sunset the existing OpenTracing and OpenCensus SIGs by **November, 2019**. 
+
+Parity can be defined as the following milestones:
+* A set of interfaces which define the OpenTelemetry Specification in a given programming language.
+* An SDK which can propagate context and record data in common formats.
+* Backwards compatibility with OpenTracing and OpenCensus.
+* Metadata helpers for recording common operations defined in the OpenTelemetry data model.
+* Tests which provide evidence of interoperability.
+* Benchmarks which provide evidence of expected resource utilization.
+
+### SPEC: Define OpenTelemetry Interfaces
+* Metrics interface
+* Tracing interface
+* Tags interface
+
+### SDK: Propagate Common Trace Contexts
+* Propagate context in Trace-Context format
+* Propagate context in custom formats
+
+### SDK: Export Common Data Formats
+* Export data in OpenTelemetry format
+* Export data in custom format
```

Comment:
this also belongs to spec as we need to define two or three "required" OSS backends with the definition on HOW to export there (basically conversion model)

Reply:
Agreed

Reply:
So again. I'd simply refer to the spec. Saing "support as a minimum things listed in a spec for the release 1.0"

Reply:
Ok, sounds good. I like the idea of the spec making it clear what is v1.0.